adam-core/src/main/scala/org/bdgenomics/adam/rdd/variation/VariationRDDFunctions.scala

```diff
@@ -110,4 +110,5 @@ class GenotypeRDDFunctions(rdd: RDD[Genotype]) extends Serializable with Logging
         rec.getVariant.getEnd > query.start
     rdd.filter(overlapsQuery)
   }
-}
+
```

Comment:
This is just a whitespace diff; can you change this back to the old version. It's preferable to not have space between the two braces, and to have the EOF after a newline.